### ISSUE TITLE
Allgemeines zu MUSE : verspätete DOIs, JSTOR Alternative

### ISSUE THREAD

**Issue by IxKo** [ubtue/DatenProbleme#1581]
### A
**Ausführliche Problembeschreibung A** 
Bei vielen MUSE-Zss wird der **DOI** erst Tage später auf der Website erzeugt. Bei welchen Zss genau dies zutrifft und wie viele Tage genau sie verzögert erscheinen, kann ich nicht beurteilen, wahrscheinlich maximal 5 Tage.
Bei Z 555,  Z 1194 und Z 1531 scheint es verzögert zu sein und sicher noch bei weiteren.

Manche MUSE-Zss scheinen auch gar keinen DOI zu besitzen.

**Offenbare Lösungen**
Das Harvesten und Einspielen der MUSE-Zss sollte daher um 5 Tage verzögert werden. Ist das möglich?
Falls ja, ist noch abzuklären, ob es grundsätzlich bei allen MUSE-Zss gemacht werden soll, oder nur bei denen, wo wir feststellen können, dass der DOI verspätet erzeugt wird.

### B
**Ausführliche Problembeschreibung B** 
Teilweise gibt es die alternative Katalogisierungsquelle **JSTOR**. Sollte diese nicht auch bei zota bevorzugt werden?
JSTOR bietet zudem in den meisten Fällen einen DOI an.
Falls ja, müsste ein anderes Verfahren genutzt werden, da JSTOR keinen RSS anbietet. Außerdem müssten manche MUSE-Zss aus dem zota-Live-Betrieb wieder herausgenommen werden.

**Comment by IxKo** [comment]
Im Meeting besprochen:
A
Eine neue QA-Lösung soll pro Zs einstellbar sein, die dann bewirkt, dass vor dem Einspielen auf Vorhandensein eines DOI geprüft wird.

B
Nur wenn es eine Quelle seitens des Verlages gäbe, würde diese bevorzugt werden.

**Comment by jriedl** [comment]
Setzen von 024a auf required in den QA-Settings für die in Frage stehenden Zeitschriften sollte das gewünschte Verhalten liefern..

**Comment by IxKo** [comment]
Dann landen die XML-Daten ohne DOI in den Error-Files und die Zs wird auf gelb gesetzt.
Müssen dann alle gelben Datensätze manuell resettet werden, damit sie nochmals mit DOI geharvestet werden?

**Comment by jriedl** [comment]
Nein, sie gelten dann als nicht korrekt heruntergeladen und werden beim nächsten Durchlauf erneut geharvestet. 
"Gelbe" Artikel, bei denen klar ist, dass sie dauerhaft inkorrekt sind, könnten auf "deliver manually gesetzt" werden, um dauerhaftes Wiederherunterladen zu vermeiden.

**Comment by IxKo** [comment]
Habe die QA-Settings nun bei folgenden Zss angepasst:
 Z 555 und Z 1531 (noch im Test)
 Z 1194 (bereits live) Z 1531

**Comment by IxKo** [comment]
Statusbericht:
Es scheint sehr lange zu dauern bis die DOIs erzeugt werden.
Bei Z 657 sind die Artikel ohne DOI schon seit mindestens 2.9. in den Errors, weil der DOI noch nicht vorhanden ist.
Es wird weiterhin beobachtet.

**Comment by IxKo** [comment]
Im heutigen Meeting besprochen:
Da, wo Muse Probleme macht und bei den übrigen im Test, soll - wenn vorhanden - auf JSTOR umgestellt werden. 
JSTOR wird im Crawling-Verfahren laufen. 
Klärung wo (p_zot2?) und wie (Link-Struktur?) der Einstiegs-Link in Zeder hinterlegt ist mit helenanebel und jriedl. 
Wie/wo die Art des Verfahrens (RSS, Crawling, ...) festgelegt wird, klärt jriedl.

**Comment by jriedl** [comment]
Für die erste Gruppe habe ich nun die Zeder-Anpassungen in die `zotero_harvester.conf` synchronisiert, die entsprechenden Regex-Ausdrücke hinzugefügt und das auf nu eingespielt, es ist also ab sofort im Interface testbar. 

Im Bezug auf die Feldbelegung und Auswertung kann ich nur den status quo beschrieben, warum dieser Ansatz so gewählt wurde, entzieht sich meiner Kenntnis. Hier könnten wir ja noch einmal überlegen, ob es ohne großen Pflege- und Anpassungsaufwand für alle Seiten eine Alternativlösung gibt.

**Comment by IxKo** [comment]
Die DOI-Verzögerung kann durch QA-Anpassungen umgangen werden, sodass die Einträge zwar später, aber mit DOI eingespielt werden.
Die JSTOR-Alternativen scheinen sich doch nicht als aktuell herausgestellt zu haben. Teilweise gab es auch in der Zwischenzeit einen Plattformwechsel.
